install.sh

```diff
@@ -59,5 +59,7 @@ else
 	echo "Manually add the directory to your \$HOME/$shell_profile (or similar)"
 	echo "  export DENO_INSTALL=\"$deno_install\""
 	echo "  export PATH=\"\$DENO_INSTALL/bin:\$PATH\""
+	echo "After you have added the directories, run the following command:"
```

Comment:
```suggestion
	echo "After you have added the directories, reload your profile:"
```
Might be better to give a reason for the command.

Reply:
About the code, I guess it's not the best option because it's not persistent and it will only work for the current session inside your terminal. Where inserting it into the `.bashrc` is persistent.